src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -0,0 +1,225 @@
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--hover--BorderBottomColor);
+    }
+  }
+
+  &:active,
+  &.pf-m-active {
+    &::before {
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--active--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-options-menu__toggle--active--BorderBottomWidth);
+    }
+  }
+}
+
+.pf-c-options-menu__toggle {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: var(--pf-c-options-menu__toggle--MinWidth);
+  max-width: 100%;
+  padding-left: var(--pf-c-options-menu__toggle--PaddingLeft);
+  border: none;
+
+  &:not(.pf-m-text) {
+    padding-top: var(--pf-c-options-menu__toggle--PaddingTop);
+    padding-right: var(--pf-c-options-menu__toggle--PaddingRight);
+    padding-bottom: var(--pf-c-options-menu__toggle--PaddingBottom);
+  }
+
+  .pf-c-options-menu.pf-m-expanded > & {
+    &::before {
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--expanded--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-options-menu__toggle--expanded--BorderBottomWidth);
+    }
+  }
+
+  &.pf-m-plain {
+    &:not(.pf-m-text) {
+      color: var(--pf-c-options-menu__toggle--m-plain--Color);
+    }
+
+    &::before {
+      border-color: var(--pf-c-options-menu__toggle--m-plain--BorderColor);
+    }
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    .pf-c-options-menu.pf-m-expanded > & {
+      --pf-c-options-menu__toggle--m-plain--Color: var(--pf-c-options-menu__toggle--m-plain--hover--Color);
+
+      &::before {
+        border-color: var(--pf-c-options-menu__toggle--m-plain--BorderColor);
+      }
+    }
+  }
+}
+
+.pf-c-options-menu__toggle-button {
+  padding: var(--pf-c-options-menu__toggle-button--PaddingTop) var(--pf-c-options-menu__toggle-button--PaddingRight) var(--pf-c-options-menu__toggle-button--PaddingBottom) var(--pf-c-options-menu__toggle-button--PaddingLeft);
+  border: 0;
+}
+
+.pf-c-options-menu__toggle-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.pf-c-options-menu__toggle-icon {
+  margin-left: var(--pf-c-options-menu__toggle-icon--MarginLeft);
+
+  .pf-m-top.pf-m-expanded > & {
+    transform: rotate(-180deg);
+  }
+}
+
+.pf-c-options-menu__menu {
+  position: absolute;
+  top: var(--pf-c-options-menu__menu--Top);
+  z-index: var(--pf-c-options-menu__menu--ZIndex);
+  min-width: 100%;
+  padding-top: var(--pf-c-options-menu__menu--PaddingTop);
+  padding-bottom: var(--pf-c-options-menu__menu--PaddingBottom);
+  background-color: var(--pf-c-options-menu__menu--BackgroundColor);
+  background-clip: padding-box;
+  border: var(--pf-c-options-menu__menu--BorderWidth) solid transparent; // this is for high contrast mode in windows
+  box-shadow: var(--pf-c-options-menu__menu--BoxShadow);
+
+  &.pf-m-align-right {
+    right: 0;
+  }
+
+  .pf-c-options-menu.pf-m-top & {
```

Comment:
for pf-m-top modifier, should the arrow be facing upwards towards the menu, I see this in the dropdown component